docs/example.ipynb

```diff
@@ -49,14 +49,117 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 16,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
     "data_path = '../tests/ga_metrics_test_data.csv'\n",
     "data = pd.read_csv(data_path)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example of `stat_summary`  \n",
```

Comment:
I think your original subtile format is better which is `function` - Description. Could you change it back and change Tang's part as well? You could copy description from README.md